.circleci/config.yml

```diff
@@ -247,7 +248,9 @@ jobs:
             mkdir /logs
             . /opt/conda/etc/profile.d/conda.sh
             # Install
-            mamba env create
+            mamba --version
+            mamba install -c conda-forge "mamba>=2"  # mamba<2 can't solve env
+            mamba env create -n esmvaltool -f environment.yml
```

Comment:
Use the same environment here as on readthedocs and skip the mamba upgrade step to make it go faster?

Reply:
ah snap! A good one - to use it for Circle's docs too